Smartsheet.Net.Standard/Http/SmartsheetHttpClient.cs

```diff
@@ -1827,5 +1827,36 @@ public async Task<IEnumerable<Share>> ShareSight(long? sightId, IEnumerable<Shar
         }
         
         #endregion
+
+        #region Cell History
+
+        public async Task<IEnumerable<CellHistory>> GetCellHistory(long? sheetId, long? rowId, long? columnId, IEnumerable<CellInclusions> includes = null, string accessToken = null)
+        {
+            includes = includes ?? new List<CellInclusions>();
+            
+            if (sheetId == null)
+            {
+                throw new Exception("Sheet ID cannot be null");
+            }
+
+            if (rowId == null)
+            {
+                throw new Exception("Row ID cannot be null");
+            }
+
+            if (columnId == null)
+            {
+                throw new Exception("Column ID cannot be null");
+            }
+
+            var response = await this.ExecuteRequest<IndexResultResponse<CellHistory>, CellHistory>(HttpVerb.GET,
+                string.Format("sheets/{0}/rows/{1}/columns/{2}/history?include=" + string.Join(",", includes), sheetId, rowId, columnId), null,
```

Comment:
Sadly, the includes won't work like this because the include options are case-sensitive and the enum options are Pascal-cased, but the include flags are camel-cased.

Reply:
So there's two options:
1. Use camel-case naming for the enum options (which goes against naming conventions)
2. Convert the string values to camel-case when creating the include string.

I'm actually not sure which I prefer. While I like to respect naming conventions, in this case it would create more hassle than it's worth and requires knowledge about the API that the values _should_ be camel-cased. Let me think for a moment.

Reply:
Decided on option 2.
`string.Join(",", includes.Select(i => i.ToString().ToCamelCase()))`